schemas/gateway-schema.json

```diff
@@ -0,0 +1,216 @@
+                                }
+                            }
+                        },
+                        "x509": {
+                            "type": "object",
+                            "description": "Client certificate authentication support.",
+                            "additionalProperties": false,
+                            "properties": {
+                                "enabled": {
+                                    "type": "boolean",
+                                    "description": "Enable if client certificate should be considered as a source of authentication.",
+                                    "default": false
+                                },
+                                "externalMapperUrl": {
+                                    "type": "string",
+                                    "description": "URL of the service where certificate will be mapped to user. ZSS is used if no value is provided."
+                                },
+                                "externalMapperUser": {
+                                    "type": "string",
+                                    "description": "User that has permission to do such mapping. Zowe user is used if no value is provided."
+                                }
+                            }
+                        }
+                    }
+                },
+                "service": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "allowEncodedSlashes": {
+                            "type": "boolean",
+                            "description": "Allow URLs on gateway to contain encoded slashes.",
+                            "default": true
+                        },
+                        "corsEnabled": {
+                            "type": "boolean",
+                            "description": "Allow CORS on gateway.",
+                            "default": true
+                        }
+                    }
+                }
+            }
+        },
+        "server": {
+            "type": "object",
+            "description": "Configure gateway server properties.",
+            "additionalProperties": false,
+            "properties": {
+                "internal": {
+                    "type": "object",
+                    "description": "It is possible to enable another gateway port for internal communication.",
+                    "additionalProperties": false,
+                    "properties": {
+                        "enabled": {
+                            "type": "boolean",
+                            "description": "Should internal port be enabled?",
+                            "default": false
+                        },
+                        "port": {
+                            "$ref": "#/$defs/port",
+                            "default": 10017
```

Comment:
other default ports are 755X

Reply:
we have this value as default for internal port, is it different for zowe installation?

Reply:
example-zowe.yaml has 7550